jib-maven-plugin/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 ### Fixed
+- Fixed `NullPointerException` during preconditions validation (in Java 9+) when checking for presence of `null` value in ImmutableCollection.([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```

Comment:
I think we can skip Maven, as I don't think this occurs in Maven.

Reply:
Ah you're right, good catch.